src/index.js

```diff
@@ -42,4 +35,31 @@ const reload = () => {
   }
 };
 
-window.onload = reload();
\ No newline at end of file
+window.onload = reload();
+
+const task = document.getElementById('add-item');
+const addNewBtn = document.getElementById('add-new-btn');
+addNewBtn.addEventListener('click', () => {
+  listArr.push(addNew(task.value, listArr.length));
+  const newStorage = JSON.stringify(listArr);
+  localStorage.setItem('toDoList', newStorage);
+  task.value = '';
+  reload();
+});
+
+const clrBtn = document.getElementById('clear-all');
+clrBtn.addEventListener('click', () => {
+  for (let i = 0; i < listArr.length; i += 1) {
+    if (listArr[i].completed) {
+      listArr.splice(i, 1);
+      listArr.forEach((task) => {
+        if (task.index > i) {
```

Comment:
- [ ] Entries do no match position of the index entered. Kindly ensure that the property index is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal to 5).

**Sample of Locally stored data** 
![image](https://user-images.githubusercontent.com/12745474/130046903-5708bbdb-edd4-44ab-b3a8-1763c0f1b1dd.png)

Reply:
The indexes of the to-do list used to start from 0, now they are starting from 1. Requested change completed.

> NOTE: please notice this is not the local storage of my project. Mine stores information as `toDoList`, not `todoList`. Also, mine doesn't use the key `id` but `index`. Still, I had the issue of the indexes starting from 0 instead of 1.